axelor-account/src/main/resources/views/PaymentVoucher.xml

```diff
@@ -55,7 +55,7 @@
       <panel hidden="true" showIf="statusSelect == 2" itemSpan="12">
         <field name="valueForCollectionMove" form-view="move-form" grid-view="move-grid"
           hidden="true"
-          showIf="!bankEntryGenWithoutValEntryCollectionOk &amp;&amp; paymentMode.inOutSelect == 1 &amp;&amp; paymentMode.accountingTriggerSelect == 5"/>
+          showIf="!bankEntryGenWithoutValEntryCollectionOk &amp;&amp; paymentMode &amp;&amp; paymentMode.inOutSelect == 1 &amp;&amp; paymentMode.accountingTriggerSelect == 5"/>
```

Comment:
This is a JavaScript expression, don't need to check if paymentMode is not null
It's the same for the following showIf and validIf you changed, please remove these changes.